cocotb/vendor/find_libpython/__init__.py

```diff
@@ -0,0 +1,383 @@
+"""
```

Comment:
When installed from zip/sdist on centos 7 I get this:
```
Traceback (most recent call last):
  File "/usr/local/bin/cocotb-config", line 11, in <module>
    load_entry_point('cocotb==1.5.0.dev0', 'console_scripts', 'cocotb-config')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 476, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2700, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2318, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib64/python3.6/site-packages/cocotb/config.py", line 46, in <module>
    import cocotb.vendor.find_libpython as find_libpython
ModuleNotFoundError: No module named 'cocotb.vendor'
```

Reply:
Added `cocotb/vendor/__init__.py`, which solves the problem.